src/components/ui/tabs.tsx

```diff
@@ -0,0 +1,46 @@
+'use client';
+import { forwardRef, ElementRef, ComponentPropsWithoutRef } from 'react';
+import { Root, List, Trigger, Content } from '@radix-ui/react-tabs';
+import { cn } from '@/lib/utils';
+
+const Tabs = Root;
+
+const TabsList = forwardRef<
+  ElementRef<typeof List>,
+  ComponentPropsWithoutRef<typeof List>
+>(({ className, ...props }, ref) => (
+  <List
+    ref={ref}
+    className={cn(
+      'flex w-full h-12 items-center justify-between rounded-xl bg-primary p-1.5 text-white',
```

Comment:
Could you modify the text color? It is set at `gray-50` on Figma.
<img width="441" alt="image" src="https://github.com/Tascurator/tascurator-frontend/assets/144519746/171dd4e2-e200-4f9c-ad18-4ba5ba4ab367">

Reply:
In the meeting with the designers, we decided to use `text-white`.